package/components/add-another/_add-another.scss

```diff
@@ -15,13 +15,15 @@
   }
 
 
-  .hmcts-add-another__item legend {
+  .hmcts-add-another__title {
+    float: left;
```

Comment:
This commit involves fixes for add another and not just progress bar. Don't do this in future.